widget/textgrid.go

```diff
@@ -109,20 +108,12 @@ func (t *TextGrid) SetText(text string) {
 	t.Refresh()
 }
 
-// TabWidth either returns the set tab width or if not set the returns the DefaultTabWidth
-func (t *TextGrid) TabWidth() int {
-	if t.tabWidth == 0 {
+// tabWidth either returns the set tab width or if not set the returns the DefaultTabWidth
+func (t *TextGrid) tabWidth() int {
```

Comment:
Sorry to be picky, but now this is unexported it belongs lower in the file as we group exported alphabetically followed by unexported alphabetically (for each tupe) - more at https://github.com/fyne-io/fyne/wiki/Contributing.

Reply:
No worries